### DISCUSSION TITLE
Automation Manager entries not working

### DISCUSSION THREAD

**Discussion by Runningman-work** [amidaware/tacticalrmm#1562] [General]
We have the following prerequisites:
- Proxmox LXC container with debian
- traditional install on container
- letsencrypt certificates
- Windows 10 clients

First I run the firefox upgrade in a single computer task, that worked.
Next I tried to upgrade the firefox packages within an automation manager entry with a task, but the task is never run. I set "run once" and a time in the near future.

"See status" on the automated task always shows "There are no agents in this policy" and there are also no relations in the automation, but I have an exclude list there. checked the exlcude list it's correct, there should be agents to do the job.

what am I doing wrong ?

**Comment by silversword411** [comment]
You haven't provided enough information to diagnose.

https://docs.tacticalrmm.com/support/#commercial-support

**Reply by Runningman-work** [reply]
What is missing ? I will post the infos...

**Reply by wh1te909** [reply]
start by filling out the [bug report template](https://github.com/amidaware/tacticalrmm/blob/develop/.github/ISSUE_TEMPLATE/bug_report.md) and then add the full steps for us to reproduce, so that would mean screenshots of your policies, automated task, scripts etc. screenshots of everything I need to replicate the same thing you're trying to do to see if I can reproduce it

**Reply by Runningman-work** [reply]
thanks for your help, found the issue. I had to assign the automation policy to a client or site.
